cli/connect/commands.go

```diff
@@ -361,6 +361,19 @@ func setTableColumnWidthMaxFunc(console *Console,
 	return "", nil
 }
 
+// setDelimiterMarker apply delimiter to the Console object.
+func setDelimiterMarker(console *Console, cmd string, args []string) (string, error) {
+	switch len(args) {
+	case 0:
+		console.delimiter = ""
+	case 1:
+		console.delimiter = args[0]
+	default:
+		return "", fmt.Errorf("the command expects zero or single argument")
+	}
+	return "---\n...\n", nil
```

Comment:
All `tt` commands return an empty output. So I suggest to follow the pattern since it does not break a table output:
```
localhost:3013> \set language sql
localhost:3013> \set delimiter ;
---
...

localhost:3013> \set output ttable
localhost:3013> \set delimiter ;
---
...
```

```suggestion
	return "", nil
```

Reply:
Done.